src/player.js

```diff
@@ -227,6 +267,90 @@ export default class Player extends FakeEventTarget {
   }
 
   /**
+   * Sets style attributes for text tracks.
+   * @param {Object} style - text styling settings
+   * @returns {void}
+   */
+  setTextStyle(style: TextStyle): void {
+    let element = Utils.Dom.getElementById(SUBTITLES_STYLE_NAME);
+    if (!element) {
+      element = Utils.Dom.createElement('style');
+      Utils.Dom.setAttribute(element, 'id', SUBTITLES_STYLE_NAME);
+      Utils.Dom.appendChild(document.head, element);
+    }
+    let sheet = element.sheet;
+
+    while (sheet.cssRules.length) {
+      sheet.deleteRule(0);
+    }
+
+    if (this._config.playback.useNativeTextTrack) {
+      sheet.insertRule(`video.${ENGINE_CLASS_NAME}::cue { ${style.toCSS()} }`, 0);
```

Comment:
is it valid in ie11?

Reply:
check if `this._config.playback` and `style` is exists and has `toCSS()` method

Reply:
it's not valid on ie11, but we use it only on iOS and the actual setup is done in application level that should check when it wants to use it and then just set `useNativeTextTrack: true`, otherwise use the DIV with CSS styling